src/RequestFeeder.ts

```diff
@@ -0,0 +1,65 @@
+import { Wallet } from "solray";
```

Comment:
RequestFeeder seems like a bad class name. It's not a "feeder".

Maybe just call it Requester or RoundRequester?